manifests/init.pp

```diff
@@ -19,6 +19,7 @@
   $service_status     = $xinetd::params::service_status,
   $service_hasrestart = $xinetd::params::service_hasrestart,
   $service_hasstatus  = $xinetd::params::service_hasstatus,
+  $manage_confdir     = false,
```

Comment:
Could you rename this to `purge_confdir`? Because even if this is false, it is still being managed.